### DISCUSSION TITLE
JLD and JLD2

### DISCUSSION THREAD

**Discussion by GuillaumeSRichard** [slimgroup/JUDI.jl#79] [General]
Hi

I am using the GPU docker image judi-gpu:1.0.
The script example I am running calls module JLD ("using JLD") but JLD seems to have been replaced in this image by a newer version called JLD2. 

If I substitute "use JLD" with "use JLD2", it passes, but then I get an error later when running the julia binary file bp_synthetic_2004_true_velocity.jld, which is downloaded and run by my script. This binary file seems to have JLD hardcoded in it.

If I try to alias JLD2 as JLD (which should be a legal julia command according to the doc) with "using JLD2 as JLD", I get some syntax error.

Thanks.

Giuillaume

**Comment by mloubout** [comment]
Hi.
Both JLD and JLD2 are valid packages. I may have only installed JLD2 but you can easily add JLD with `Pkg.add("JLD")` if you need it

**Reply by mloubout** [reply]
IF you have both package loaded, use `JLD.@load` or `JLD2.@load` to specify which one to use as well.
